plugins/algoltek-usb/fu-algoltek-usb.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2023 Algoltek, Inc.
+// SPDX-License-Identifier: LGPL-2.1+
+
+#[derive(Getters,ValidateBytes)]
+struct AlgoltekProductIdentity {
+    header_len: u8,
+    header: u64le == 0x4B45544C4F474C41, // 'A' 'L' 'G' 'O' 'L' 'T' 'E' 'K'
+    product_name_len: u8,
+    product_name: [char; 16],
+    version_len: u8,
+    version: [char; 48],
+}
+
+enum AlgoltekCmd {
+    RDR = 0x06,
+    WRR,
```

Comment:
I think these need to be `Rdr`, `Wrr` etc.

Reply:
Updated!